src/TestUtils/GcloudWrapper.php

```diff
@@ -112,16 +112,34 @@ public function setDir($dir)
     /**
      * Deploy the app to the Google Cloud Platform.
      *
-     * @param string $targets optional The yaml files for deployments.
-     * @param bool $promote optional true if you want to promote the new app.
-     * @param int $retries optional Number of retries upon failure.
+     * @param array $options list of options
+     *      $targets string The yaml files for deployments.
+     *      $promote bool True if you want to promote the new app.
+     *      $retries int Number of retries upon failure.
+     *      $version string Run gcloud using "alpha" or "beta" version of gcloud commands
```

Comment:
Take it a grain of salt again, but `$version` is little bit confusing with the appengine version, which in this case stored in the instance variable. I'd use `$gcloud_channel`, but do you have any other good idea?

Reply:
Good point on the instance option. I took a look and couldn't find the word "channel" in use anywhere, which is why I didn't want to use that. Also, `gcloud` seems redundant.

I could use `channel`, or `release_version`. What do you think?

Reply:
`release_version` works for me. Thanks!